docs/devel/testing.md

```diff
@@ -174,55 +174,7 @@ hack/test-integration.sh  # Run all integration tests.
 
 ## End-to-End tests
 
-* e2e tests build kubernetes and deploy a cluster of nodes.
-  - Generally on a specific cloud provider.
-* Access gcr.io images
-* Access a specific, non-latest image tag (unless testing pulling).
-* Tests may not flake due to intermittent issues.
-* Use ginko to desribe steps.
-  - See [should run a job to completion when tasks succeed](../../test/e2e/job.go)
-* Use [NewDefaultFramework](../../test/e2e/framework.go)
-  - Contains clients, namespace and auto resource cleanup
-* See [coding conventions](coding-conventions.md).
-
-### e2e test philosophy
```

Comment:
Sure we can always add back something similar if someone misses it, and I'm always for less code.

On the other hand I liked having at least one command to run tests to serve as a reminder (I pretty much never do anything without having a reference doc open reminding me how to do it):

Maybe add something like this in a future PR:

```sh
cd kubernetes
go run hack/e2e.go -v --build --up --test --down  --test_args="--ginkgo.focus=\[Conformance\]"
```